doc/changes/names.inc

```diff
@@ -66,6 +66,8 @@
 
 .. _Britta Westner: https://britta-wstnr.github.io
 
+.. _Balasubramanian T K: https://github.com/btkcodedev
+
 .. _Bruno Nicenboim: https://bnicenboim.github.io
```

Comment:
Could you move this entry up, in its alphabetical position?

Reply:
Am I correct that Balasubramanian is your surname? If so, to get correctly credited in our citations, it would be helpful if this entry were `T K Balasubramanian` (and sorted with the "T" entries)... the script that generates our citation file assumes that the surname comes last in these entries.

Reply:
I could go with btkcodedev :)

Reply:
That is also ok if that's what you prefer!